app/helpers/buyers/applications_helper.rb

```diff
@@ -3,13 +3,30 @@ module Buyers::ApplicationsHelper
   def metadata_new_app(buyer, provider)
 
     "<div id='metadata-form'
+      data-application_plans='#{ application_plans(provider) }'
+      data-default_plan='#{ default_plan(provider) }'
       data-services_contracted='#{ services_contracted(buyer) }'
       data-service_plan_contracted_for_service='#{ service_plan_contracted_for_service(buyer) }'
       data-relation_service_and_service_plans='#{ relation_service_and_service_plans(provider) }'
```

Comment:
Here's escaping badly when the there are special characters... then it fails when decoding json.